src/parser.rs

```diff
@@ -55,12 +55,20 @@ named!(priority<&str, u8>,
     )
 );
 
-fn get_tags(delim: &str, subject: &str) -> Vec<String>
-{
-    let expression = format!("(?P<space>^|[\\s]){}(?P<tag>[\\w-]+)", delim);
-    let regex = ::regex::Regex::new(&expression)
-        .unwrap();
+macro_rules! regex_tags_shared {
+    () => { "(?P<space>^|[\\s]){}(?P<tag>[\\w-]+)" }
+}
+lazy_static! {
```

Comment:
I think, it’s better to move each regexp in its function, more close where they used.

Reply:
Indeed, fixed, thanks!